### PR TITLE
Added SPAtmplt option in _get_imr_duration

### PR DESCRIPTION
**Pull request by AlexisMV**
Modified the _get_imr_duration function inside 'pnutils'  to include the approximant SPAtmplt (TaylorF2)

### CODE REVIEW

File: pycbc/pnutils.py
Comment (titodalcanton) [inline pycbc/pnutils.py:525]: Hi @AlexisMV, could you do a couple more whitespace fixes to address the codeclimate and codebeat complains:
* `m1,m2,s1z,s2z` -> `m1, m2, s1z, s2z`
* Indent with 8 spaces (two tabs)
* Remove the trailing white space after the function call
Also please rebase on master, I think this is needed after Alex's changes to the test suite.

Comment (AlexisMV) [reply pycbc/pnutils.py]: Hi @titodalcanton , thanks a lot, I was not sure what it was asking from me. I tried git rebase master and it says the branch is up to date, not sure if I need to do something else (Sorry, still learning git).

**Comment by titodalcanton** [comment]
Thanks @AlexisMV, it actually looks like rebasing is not necessary, I see the tests completed after you pushed the new commits. Codebeat is still complaining about the level of nesting, but I am not sure why, and I think that can be ignored. This looks good to me, assuming you tested it.

**Comment by AlexisMV** [comment]
Hi @titodalcanton. Yes, I tested it for different values of masses and spins and it was working fine. The output also seem to be ok, the duration given is similar to the one obtained by using the approximant IMRPhenomD.

**Review by titodalcanton — APPROVED** [review]
In that case I am happy to approve this.

**Comment by titodalcanton** [comment]
Hmm, it looks like I am no longer allowed to merge this after the change to the tests was merged. @AlexisMV could you try doing a rebase as we discussed a few days ago? That should hopefully trigger the new tests.

**Comment by titodalcanton** [comment]
Superseded by #3540 to fix the branch name.

**Pull request closed without merging**
